test-terminal-memory-e2e.js

```
@@ -0,0 +1,344 @@
+			"terminalId",
+		]
+
+		for (const record of this.terminalRecords.slice(0, 3)) {
+			// Check first 3 records
+			const payload = record.payload
+
+			console.log(`   📋 Record ID: ${record.id}`)
+
+			for (const field of expectedFields) {
+				if (payload.hasOwnProperty(field)) {
+					console.log(`   ✅ ${field}: ${JSON.stringify(payload[field]).substring(0, 50)}...`)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)